README.md

```diff
@@ -37,6 +37,14 @@ This is a totally functional Tic Tac Toe game ready to play in your computer ter
 4. Once in the main folder insert `cd bin`
 5. Here you have to insert `./main.rb` to execute the game and follow the instrucions on screen.
 
+## Testing with RSpec
+
+For professional users: RSpec tests have been included to check to correct functioning of the program.
+For accessing the RSpec tests follow the previous step 'Installation' must be completed. The you can follow these steps:
+1. Go to the main directory `TicTacToe-Ruby`
+2. Type `rspec` and press Enter. The tests will be displayed on your screen.
+- The tests are stored in the spec folder.
```

Comment:
- [ ] A person without your configurations cannot make your program work like this, the person would need to run `gem install rspec` before running `rspec`  so that it can work, please add it to your instructions :)
- _**Note:**_ If you ever want to see how others see your project, you can append `gitpod.io/` to your repo URL, this will open an "empty virtual computer on the cloud" where you can check out your project, like this: https://gitpod.io/github.com/Daniduran-dev/TicTacToe-Ruby/tree/testing

Reply:
Nice!